test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Microsoft.Testing.Platform.Acceptance.IntegrationTests": {
       "commandName": "Project",
-      "commandLineArgs": "--treenode-filter /*/*/*/**",
+      // "commandLineArgs": "--treenode-filter /*/*/*/**",
```

Comment:
Is this change intended?

Reply:
Yes! The filter isn't supported by MSTest so when you try to run the project it fails. I need to double check this is done everywhere!